lib/commands/connect/sf-auth.js

```diff
@@ -20,9 +20,9 @@ function callbackServer() {
 
 module.exports = {
   topic: 'connect',
-  command: 'auth',
-  description: 'get Salesforce authorization URL',
-  help: 'return an URL to authorize a connection to a Salesforce Org',
+  command: 'sf:auth',
+  description: 'Authorize access to Salesforce for your connection',
+  help: 'Provides a URL to authorize a connection to a Salesforce Org',
```

Comment:
Well, technically it does both (shows a URL and opens a browser to said URL), and I was just trying to find a way to make the help different from the description, so it wasn't just a duplication. But yeah, displaying the URL isn't really the purpose, so maybe that doesn't make sense.